lib/resource/InstanceResource.js

```diff
@@ -55,4 +57,51 @@ InstanceResource.prototype.delete = function deleteResource(callback) {
   this.dataStore.deleteResource(this, callback);
 };
 
+InstanceResource.prototype.invalidateCache = function invalidateResource(callback) {
```

Comment:
I actually first named it just `invalidate()` but felt that was kind of weak/unclear and could cause confusion. So decided to go with `invalidateCache()` because it was clearer. Do you still think that we should just have `invalidate` or is `invalidateCache()` good?